test/unit/calculators/student_finance_calculator_test.rb

```diff
@@ -39,6 +39,33 @@ class StudentFinanceCalculatorTest < ActiveSupport::TestCase
         assert_equal 'uk-full-time', calculator.course_type
       end
 
+      context "childcare_grant" do
+        context "for one child" do
+          context "in 2016-2017" do
+            should "be 155.24" do
+              calculator = StudentFinanceCalculator.new(
+                course_start: "2016-2017",
+                household_income: 25_000,
+                residence: :unused_variable
```

Comment:
This was pre the refactor of the StudentFinanceCalculator.. I moved the commit order around to help understand the intent here..